### ISSUE TITLE
Cannot post to chat anymore

### ISSUE THREAD

**Issue by Mansarde** [keybase/client#20483]
Recently whenever I try to post anywhere there is a blue padlock next to my message that moves back and forth. But nothing happens and my message doesn't get posted.
I also tried PC restarts and refreshing my IP, but when I go back into the channel or personal chat, then my messages still have the blue padlock icon on them.
What does that mean and how can I fix it?

my log id: ad94fb059a89da2c457edf1c

**Comment by Mansarde** [comment]
Since the OP I have nuked the DB which didn't resolve the problem.
Then I completely removed any trave from my computer, even removed it as a device from the chain.
After re-installing it, I could send a test message in chat again.
Since then I did nothing with keybase except to watch conversation.

Now, two days later, I try to send something again, but the blue padlock is back again!
And after a while it stops moving and becomes a grey opened padlock.

I would be glad to simply know what the padlock is doing and how it could fail to send my messages.

Here's a new log id from the fresh install: 6cbf7dfd638cd3c68c88701c

**Comment by Mansarde** [comment]
Ok, only killing the Keybase app doesn't help, but after killing every Keybase service-process and then restarting the Keybase app made it possible to post again.
So it seems the problem occurs only after a while.
I also noticed that the Logs get pretty big (~130 MB) and contain a lot of debug messages despite me not having Debug Logging enabled in the app.

But that would be another issue. Closing this for now as I have a work-around for the intermittent non-working chat.

I'm sure you guys currently have your hands full with the Stellar account abuse, so the less issue-posts the better. Hope the situation will get better soon for you. Wish you the best! :)
